config/database.yml

```diff
@@ -1,25 +1,23 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql 
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  encoding: unicode
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: member_portal_development 
```

Comment:
Might want to make the database names consistent across the environments.  Use all hyphens or all underscores.